testnets/qwoyntestnet/chain.json

```diff
@@ -5,7 +5,7 @@
   "network_type": "testnet",
   "website": "https://qwoyn.studio/",
   "pretty_name": "Qwoyn",
-  "chain_id": "higgs-boson-1",
```

Comment:
also, it would be best to retain higgs-boson-1 as chain_name /qwoyntestnet2/
Improvements to chain_name management are coming, but for now this allows us to retain historical data.

Reply:
If needed, I create another PR that adds the old chain data as qwoyntestnet2 so that this PR can proceed

Reply:
https://github.com/cosmic-horizon/chain-registry/pull/2

Reply:
Any update there?